draft-barnes-tls-pake.md

```diff
@@ -154,7 +152,7 @@ selected, and the server's first message in the PAKE protocol.
 
 Use of PAKE authenication is compatible with standard
 certificate-based authentication of both clients and servers.  If a
-server includes n `pake` extension in its ServerHello, it may still
+server includes an `pake` extension in its ServerHello, it may still
```

Comment:
I think this would be "a", since pake" is pronounced starting with a consonant sound.